src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -251,19 +251,18 @@ export default class UploadDetachedFilesPage extends React.Component {
 				if (this.isUnmounted) {
 					return;
 				}
-				if(response.publish_status=='unpublished'){
-					this.setState({submit: true})
-				}
-				else if(response.publish_status=='published'){
-					this.setState({submit: false})
+				var submission = true;
+				if(response.publish_status=='published'){
```

Comment:
You want to check only for "unpublished" not for "published" since there are multiple versions of "published" but only one unpublished.